magento-research/pwa-studio
# [feature]: refactor edit steps out of redux and into local checkout state

## Description

This is a followup to #1309. The original refactor left much of the original redux piping in place. This PR shows how we can remove parts of it relatively painlessly. Here we are migrating the state of what form is being edited, ie payment, address or shipping, into local form state. 

I also removed some unnecessary props and added some safety checks/state to make sure we only enable the submission of billing information when appropriate.

## Related Issue





Closes #1405 

## Verification Steps

1. Open checkout and work through the steps.
2. You should be able to open and close each step in no particular order, and no errors should occur.

## Screenshots / Screen Captures (if appropriate)

## Proposed Labels for Change Type/Package




## Checklist:
